SimpleServlet/src/main/java/com/skankhunt220/service/UserService.java

```diff
@@ -1,51 +1,32 @@
 package main.java.com.skankhunt220.service;
 
-import static com.mongodb.client.model.Filters.eq;
-
 import org.bson.Document;
-import org.bson.types.ObjectId;
-
-import com.mongodb.client.MongoDatabase;
-import com.mongodb.MongoClient;
-import com.mongodb.client.MongoCollection;
 
 import main.java.com.skankhunt220.entities.User;
+import main.java.com.skankhunt220.repository.UserRepository;
 
 public class UserService {
 
-	private User user;
-	private MongoClient mc;
-	private MongoDatabase database;
-	private MongoCollection<Document> collection;
-
-	public UserService(User user) {
-		this.user = user;
-		mc = new MongoClient("localhost", 27017);
-		database = mc.getDatabase("UserDB");
-		collection = database.getCollection("users");
+	private UserRepository userRepository;
+	
+	public UserService() {
+		userRepository = new UserRepository();
 	}
 
-	public void Create() {
-		collection.insertOne(new Document("_id", new ObjectId()).append("firstName", user.getFirstName())
-				.append("middleName", user.getMiddleName()).append("lastName", user.getLastName()));
-		mc.close();
+	public void createUser(User user) {
+		userRepository.Create(user);
 	}
 
-	public Document Read() {
-		Document myDoc = collection.find(eq("_id", new ObjectId(user.getId()))).first();
-		mc.close();
+	public Document readUser(String userId) {
+		Document myDoc = userRepository.Read(userId);
```

Comment:
I think it's a bad idea to create variables which are used only once (unless it's absolutely needed). Rewrite it like:
`return userRepository.Read(userId);`